tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -1664,6 +1664,47 @@ public function data_next_tag_ignores_script_tag_contents() {
 		);
 	}
 
+	/**
+	 * Invalid tag names are comments on tag closers.
+	 *
+	 * See https://html.spec.whatwg.org/#parse-error-invalid-first-character-of-tag-name
+	 *
+	 * @ticket 58007
+	 * @dataProvider data_next_tag_ignores_invalid_first_character_of_tag_name_comments
```

Comment:
```suggestion
	 * @ticket 58007
	 *
	 * @dataProvider data_next_tag_ignores_invalid_first_character_of_tag_name_comments
```
`@ticket` and `@dataProvider` should be separated by an empty line

Reply:
split in ddcb554583